1-Authentication/2-sign-in-b2c/README.md

```diff
@@ -1,24 +1,44 @@
+---
+page_type: sample
+name: Angular single-page application using MSAL Angular to sign-in users against Azure AD B2C
+description: Angular single-page application using MSAL Angular to sign-in users against Azure AD B2C
+languages:
+ -  javascript
+ -  typescript
+products:
+ - azure-active-directory-b2c
+ - msal-js
+ - msal-angular
+urlFragment: ms-identity-javascript-angular-tutorial
+extensions:
+- services: ms-identity
+- platform: JavaScript
+- endpoint: AAD v2.0
+- level: 100
+- client: Angular SPA
+- service: 
+---
+
 # Angular single-page application using MSAL Angular to sign-in users against Azure AD B2C
 
- 1. [Overview](#overview)
- 1. [Scenario](#scenario)
- 1. [Contents](#contents)
- 1. [Prerequisites](#prerequisites)
- 1. [Setup](#setup)
- 1. [Registration](#registration)
- 1. [Running the sample](#running-the-sample)
- 1. [Explore the sample](#explore-the-sample)
- 1. [About the code](#about-the-code)
- 1. [More information](#more-information)
- 1. [Community Help and Support](#community-help-and-support)
- 1. [Contributing](#contributing)
+* [Overview](#overview)
+* [Scenario](#scenario)
+* [Contents](#contents)
+* [Prerequisites](#prerequisites)
+* [Explore the sample](#explore-the-sample)
+* [Troubleshooting](#troubleshooting)
+* [About the code](#about-the-code)
+* [Next Steps](#next-steps)
+* [Contributing](#contributing)
```

Comment:
Learn more and step up the sample titles are missing.

Reply:
done